files/teleirc/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM permissionerror/teleirc
```

Comment:
Ideally, I'd like to remove this file and just use the 'official' one, let me know if something needs to be changed on the default image.

Reply:
Yep, a docker volume sounds good. The file is needed so that the needed values can be configured in `.env` instead of `config.js`.